ci/badge/gen_badge.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+##
+# Copyright (C) 2019 Samsung Electronics
+# License: LGPL-2.1
+#
+# @file gen_github_badge_svg.py
+# @brief A tool for generating a github badge image in svg format
+# @author Wook Song <wook16.song@samsung.com>
+
+import sys
+import os
+from bs4 import BeautifulSoup
+import requests
+import svgwrite
+
+##
+# @brief Get a string representing the whole html file of a given url
+# @param[in] url A valid url-string indicating a specific html file
+def get_html(url):
+    """get_html(url) -> str"""
```

Comment:
BTW, does Doxygen recognize the **"""** symbols?

Reply:
@leemgs This line is not for Doxygen but for Docstring Convention ([PEP 257](https://www.python.org/dev/peps/pep-0257/)).

Reply:
@wooksong I mean that Doxygen can be broken in case that """ statements are written for PEP 257. Please check it by running doxygen command manually.